Heart/Provider/Presentation/Controllers/ProvidersController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Heart\Provider\Presentation\Controllers;
+
+use App\Http\Controllers\Controller;
+use Heart\Provider\Application\NewAccountByProvider;
+use Heart\Provider\Domain\Enums\ProviderEnum;
+use Heart\Provider\Presentation\Requests\CreateProviderRequest;
+use Symfony\Component\HttpFoundation\Response;
+
+class ProvidersController extends Controller
+{
+    public function postProvider(
+        CreateProviderRequest $request,
+        string $provider,
+        NewAccountByProvider $action,
+    ) {
```

Comment:
```suggestion
    ): JsonResponse {
```
use Illuminate\Http\JsonResponse;